pages/authorized_page.py

```diff
@@ -15,3 +15,9 @@ def label_page_head(self):
 
     def get_page_head(self):
         return self.label_page_head().text
+
+    def current_url(self):
```

Comment:
Would it make sense to have this method in base_page.Page?

Reply:
Yes, sorry, i forget about it. Already locally merge master in this branch